setup.cfg

```diff
@@ -13,7 +13,7 @@ install_requires =
 include_package_data = True
 
 [options.package_data]
-dss = manifests.yaml
+dss = manifest_templates/*
```

Comment:
This definitely works, but it sounds like it'd be better in its own PR. You can also mention it is part of #44.